Source/Shaders/PostProcessStages/ACESTonemapping.glsl

```diff
@@ -0,0 +1,31 @@
+uniform sampler2D colorTexture;
+
+varying vec2 v_textureCoordinates;
+
+#ifdef AUTO_EXPOSURE
+uniform sampler2D autoExposure;
+#endif
+
+// See:
+//    https://knarkowicz.wordpress.com/2016/01/06/aces-filmic-tone-mapping-curve/
+
+void main()
+{
+    vec3 color = texture2D(colorTexture, v_textureCoordinates).rgb;
+
+#ifdef AUTO_EXPOSURE
+    color /= texture2D(autoExposure, vec2(0.5)).r;
+#endif
+    float g = 0.985;
```

Comment:
A thought for when auto exposure is not used here and potential for other tone mappers: is it possible to select good constants based on indoor vs. outdoor, global vs. local, satellite imagery vs. vectors, photogrammetry vs. lots of primitives, etc. - could we provide a handful of decent defaults to select from?  Or does it just vary too widely based on the frequency of whatever content an app loads?

Also, what is auto exposure on by default?  What is wrong with it and what is the path to correct it now or later?